projects/demo/src/modules/experimental/button-group-card/examples/1/index.html

```diff
@@ -0,0 +1,70 @@
+<div
+    tuiButtonGroupCard
+    tuiSurface="elevated"
+>
+    <button
+        appearance="flat"
+        iconLeft="tuiIconPlusCircleLarge"
+        tuiButton
+        tuiButtonVertical
+    >
+        Create a payment
+    </button>
+
+    <button
+        appearance="flat"
+        iconLeft="tuiIconPlusCircleLarge"
+        tuiButton
+        tuiButtonVertical
+    >
+        Pay
+        <br />
```

Comment:
let's use `&nbsp;` between `the` and `bill` instead

Reply:
is it ok?

<img width="374" alt="image" src="https://github.com/taiga-family/taiga-ui/assets/12021443/359c1ca9-fd52-4ce9-bd69-fe71e240e240">

Reply:
It should just be aligned to the top

Reply:
fixed